src/include/catalog/unused_oids

```diff
@@ -28,10 +28,22 @@ chdir $FindBin::RealBin or die "could not cd to $FindBin::RealBin: $!\n";
 use lib "$FindBin::RealBin/../../backend/catalog/";
 use Catalog;
 
+# Some of OIDs register in CBDB kernel
+my @excluded_oids = (
+    # ex.
+    # 4, 5           # excluded single OID
+    # 10000..10050   # excluded range OIDs
+
+    7047             # PAX AM ID
```

Comment:
It's highly recommend to extract the oid value from header file `#define PAX_TABLE_AM_OID 7047` in contrib/pax_storage/src/cpp/comm/pax_rel.h.

Reply:
The file `src/include/catalog/duplicate_oids` also hard-coded the oid of pax table am.

Reply:
we can impl function like `FindAllOidsFromHeaders`